src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private int ballNumber;
+
+    public Ball getRandomBall() {
+        int constant = 100;
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery
All magic numbers in your code should be constants.
Please see [this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) to learn about constant fields and their naming requirements.